wallet-confs/trittium--v4.0.2.1.conf

```diff
@@ -0,0 +1,12 @@
+server=1
+listen=1
+rpcuser=
+rpcpassword=
+ 
+rpcallowip=0.0.0.0/0
+port=31001
+rpcport=31002
+txindex=1
+
+
+
```

Comment:
I'm not sure why this `wallet-confs/trittium--v4.0.2.1.conf` file is being added. `manifest-latest.json` doesn't ever reference this config file. 😕